src/modules/app/createReduxStore.js

```diff
@@ -15,17 +15,14 @@ import { routesMap as defaultRoutesMap } from './route-configs/index'
 import onBeforeChange from './onBeforeChange'
 import queryString from 'query-string'
 import { Payload } from '@integreat-app/integreat-api-client'
-import createHistory from './createHistory'
 
 export type ActionType<T> = { type: string, payload: Payload<T> }
 
 // todo: Change type to correct State type,
 // https://blog.callstack.io/type-checking-react-and-redux-thunk-with-flow-part-2-206ce5f6e705
 const createReduxStore = (initialState: {} = {}, routesMap: RoutesMap = defaultRoutesMap): Store<any, any> => {
-  const history = createHistory()
-
-  const {reducer, middleware, enhancer} = connectRoutes(history, routesMap, {
-    onBeforeChange: onBeforeChange,
+  const {reducer, middleware, enhancer} = connectRoutes(routesMap, {
```

Comment:
Our history was responsible to handles scrolls between navigation changes. Is this still handled?